File: Pil-Pr/src/app/models/Usuario.ts


export class Usuario{

    // private Id:number;
    // private DNI:string;
    // private Nombre:string;
    // private Apellido:string;
    // private Email:string;
    // private NombreUsuario:string;
    // private Clave:string;
    // private FotoPerfil:string;
    // private FotoDNI:string;


    // constructor(id:number, dni:string, nombre:string, 
    //     apellido:string, email:string, nombreUsuario:string, clave:string, fotoPerfil:string, fotoDNI:string){
    //     this.Id = id;
    //     this.DNI = dni;
    //     this.Nombre = nombre;
    //     this.Apellido = apellido;
    //     this.Email = email;
    //     this.NombreUsuario = nombreUsuario;
    //     this.Clave = clave;
    //     this.FotoPerfil = fotoPerfil;
    //     this.FotoDNI = fotoDNI;
    // }

    constructor(
        public Id:number,
        public DNI:string,
        public Nombre:string,
        public Apellido:string,
        public Email:string,
        public NombreUsuario:string,
        public Clave:string,
        public FotoPerfil:string,
        public FotoDNI:string
    ){}
}

File: BEPilMoney/BEPilMoney/Models/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BEPilMoney.Models
{
    public class Usuario
    {
        #region Propiedades
        public int Id { get; set; }
        public string DNI { get; set; }
        public string Nombre { get; set; }
        public string Apellido { get; set; }
        public string Email { get; set; }
        public string NombreUsuario { get; set; }
        public string Clave { get; set; }
        public string FotoPerfil { get; set; }
        public string FotoDNI { get; set; }
        #endregion
    }
}